docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>I miss a mention of the Sampler, either we clarify at the beginning of the section that we are talking only about the estimator, or we add after: "<code>EstimatorV2</code> returns one expectation value estimate for each element of the broadcasted shape" something like "<code>SamplerV2</code>  doesn't accept observables so the parameter shape is always mapped to the broadcasted result" (I believe this is the case, but not 100% confident).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>You are correct, broadcasting is only relevant to the Estimator because it is the only one that accepts multiple array based inputs. I'd include "Estimator" somewhere in the first sentence of the section.</p>